src/main/webapp/app/quiz/edit/multiple-choice-question/edit-multiple-choice-question.html

```diff
@@ -0,0 +1,121 @@
+<div class="edit-mc-question" ng-hide="vm.showPreview">
+    <div class="question-options">
+        <div class="form-group question-title">
+            <input
+                class="form-control"
+                ng-model="vm.question.title"
+                placeholder="Question Title"
+            />
```

Comment:
One question (about the whole file) and the code format: Can we write tags such as <input .../> into one line?
Otherwise it gets quite cluttered and I think the other HTML code also writes them onto one line.
Feel free to use an automatic formatting tool